### PR TITLE
Add special e2e tests to PR #139

### PR DESCRIPTION
**Pull request by kieferro**

### CODE REVIEW

**Comment by ewjoachim** [comment]
Excellent idea !

File: tests/end_to_end/test_all.py [hunk 1]
```diff
@@ -124,6 +124,13 @@ def test_public_repo(
     )
     assert ":arrow_up:" in comment
 
+    # Let's merge the PR and see if everything works fine
+    gh_me("pr", "merge", "1", "--merge")
+    git("fetch")
```
Comment (ewjoachim) [inline tests/end_to_end/test_all.py:129]:
```suggestion
    git("fetch", env={"GITHUB_TOKEN": token_me})
```

File: tests/end_to_end/test_all.py [hunk 2]
```diff
@@ -259,3 +266,10 @@ def test_private_repo(
         fail_value="\n",
     )
     assert ":arrow_up:" in comment
+
+    # Let's merge the PR and see if everything works fine
+    gh_me("pr", "merge", "1", "--merge")
+    git("fetch")
```
Comment (ewjoachim) [inline tests/end_to_end/test_all.py:272]:
```suggestion
    git("fetch", env={"GITHUB_TOKEN": token_me})
```

**Comment by kieferro** [comment]
Good point, I didn't see that the fetch didn't work for the private repo

**Pull request merged**